zipline/data/minute_bars.py

```diff
@@ -265,7 +274,8 @@ def read(cls, rootdir):
 
     def __init__(
         self,
-        ohlc_ratio,
+        default_ohlc_ratio,
```

Comment:
Was trying to make the distinction from `ohlc_ratios_per_sid` clear in the naming, but if it's overkill I'd be happy to simplify it.